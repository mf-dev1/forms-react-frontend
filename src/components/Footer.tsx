import * as React from 'react';

class Footer extends React.Component {
  public render() {
    return (
      <footer>
        <div className="text-center">Footer Text</div>
      </footer>
    );
  }
}

export default Footer;
