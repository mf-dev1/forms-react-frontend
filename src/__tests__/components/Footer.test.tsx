import * as React from 'react';
import { create } from 'react-test-renderer';
import Footer from '../../components/Footer';

describe('Footer component', () => {
  test('it matches the snapshot', () => {
    const component = create(<Footer />);
    expect(component.toJSON()).toMatchSnapshot();
  });
});
