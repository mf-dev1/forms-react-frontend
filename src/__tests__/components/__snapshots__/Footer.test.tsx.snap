// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Footer component it matches the snapshot 1`] = `
<footer
  className="Footer-footer-1"
/>
`;
